app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -76,7 +76,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsCellPhones = false
         allProducts!!.forEach {
-            if (it.product.equals("Cell Phones")) {
+            if (it.product == ("Cell Phones")) {
```

Comment:
(Did this in many places): Replaces .equals with ==, per Android studio's recommendation. It is a simpler syntax
https://kotlinlang.org/docs/equality.html